robotiq_s_model_articulated_msgs/msg/SModelRobotInput.msg

```diff
@@ -0,0 +1,154 @@
+# gACT : Initialization status, echo of the rACT bit (activation bit).
```

Comment:
This is the input message that matches the one found in robotiq_s_model_control/msg.  It functionally should work the same.  I actually think this version is better and it might be better to be in it's own package to make the dependency tree smaller if you only care about gazebo and don't want to install the control package.  Assuming of course that this ever gets released as a debian.  However keeping this version and removing the one in robotiq_s_model_control will affect that package and anyone else using it, which would be disruptive.

We can let the maintainers decide how to handle it.